services/jobrunner/justfile

```diff
@@ -20,6 +20,18 @@ stop:
 restart:
     sudo systemctl restart jobrunner
 
+# Update jobrunner to the specified ref & restart
+deploy ref="origin/main":
```

Comment:
can we do this?
```suggestion
deploy ref="origin/main": update-lib migrate
```

Reply:
Did this in the script instead, as there's some ordering and error handling subtleties.